src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -83,105 +94,119 @@
                 </TextBlock>
             </StackPanel>
         </Grid>
-
-        <Grid x:Name="TaintList">
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Name="Table" Width="*"/>
-            </Grid.ColumnDefinitions>
-            <DataGrid Grid.Column="0" 
+        <Grid x:Name="Content">
```

Comment:
![image](https://user-images.githubusercontent.com/60586879/106281600-ba1f3080-623f-11eb-9ccf-f05923b4b3c3.png)

I looked into adding a yellow-ish background like in SLI, but it looked too much like a goldbar.

Reply:
Looks good.
I think it would better to have the branch name in quotes or italics - something to make it stand out more from the boiler plate text.

Reply:
I went with single quotes, we can change after dogfood feedback